src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+function toNumber(salary) {
+  return parseFloat(salary.substring(1).replaceAll(',', ''));
+}
+
+function sortList(list) {
+  const lst = document.querySelector(list);
+
+  [...lst.children]
+    .sort((a, b) => toNumber(b.dataset.salary) - toNumber(a.dataset.salary))
+    .forEach(node => lst.appendChild(node));
+}
+
+function getEmployees(list) {
```

Comment:
you don't use the argument `list`. delete it or use in query selector as in the `sortList` function

Reply:
tacking into account that you always use `ul` in querySelector you can delete it here and in `sortList` as well